### ISSUE TITLE
[MudDateWheelPicker] Scrolling through values also scrolls the page

### ISSUE THREAD

**Issue by Papa-Tori** [CodeBeamOrg/CodeBeam.MudExtensions#219]
If the page can be scrolled up (or down), scrolling through the values of the wheel will scroll the page, too.
Repro link:
https://codebeam-mudextensions.pages.dev/muddatewheelpicker

**Comment by mckaragoz** [comment]
Probably this occurs only on Firefox, right?

**Comment by Papa-Tori** [comment]
Yeah, that's right

**Comment by DylanSnel** [comment]
I tested this component on mobile on edge chrome and firefox. You dont seem to actually grab the number you scroll and then it moves the numbers on release. that is kind of confusing

**Comment by mckaragoz** [comment]
> I tested this component on mobile on edge chrome and firefox. You dont seem to actually grab the number you scroll and then it moves the numbers on release. that is kind of confusing

Yes, it doesn't a finished component yet. It uses MudSwipeArea and doesn't react on real time. It detects the swipe and after then changes value. After we added touchmove ability to MudSwipeArea we can enhance the MudWheel.
